src/integration-test/java/uk/gov/ida/integrationtest/VerifyUserAccountCreationIntegrationTest.java

```diff
@@ -50,21 +28,4 @@ protected HttpStubRule setUpMatchingService() throws Exception {
         return localMatchingService;
     }
 
-    @Test
```

Comment:
This class now doesn't have any tests - can we just remove it completely?

Reply:
Oh of course! Will remove